tic_tac_toe/game.py

```diff
@@ -87,18 +149,52 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    board = game['board']
+    current_player = get_next_turn(game)
+
+    if _board_is_full(board) or game['winner'] != None:
+        raise InvalidMovement('Game is over.')
+
+    if current_player != player:
+        raise InvalidMovement('"' + current_player + '" moves next.')
```

Comment:
Be careful with this way of concatenating string as it is not optimal. If the player choses numbers instead of 'x' and 'o' this will break:
```python
>>> current_player = 2
>>> '"' + current_player + '" moves next.'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Can't convert 'int' object to str implicitly
```
The `string.format()` method would be better for any string concatenation as it casts everything to string implicitly.